smoke/IotEdgeQuickstart/details/PartitionReceiveHandler.cs

```diff
@@ -26,6 +26,6 @@ public Task ProcessEventsAsync(IEnumerable<EventData> events)
             return Task.CompletedTask;
         }
         public Task ProcessErrorAsync(Exception error) => throw error;
-        public int MaxBatchSize { get; } = 10;
+        public int MaxBatchSize { get; set; }
```

Comment:
Who uses this setter?

Reply:
The interface requires it.

Reply:
There doesn't seem to be any reason why [they changed it to require a setter.](https://github.com/Azure/azure-event-hubs-dotnet/commit/7e6b7b9b1d01a2ebc3bfe8513cb43efe10fddf42) The `IPartitionReceiver` impl in the SDK only gets its value.